cmd/context/create.go

```diff
@@ -227,6 +240,18 @@ func (c *ContextCommand) initOktetoContext(ctx context.Context, ctxOptions *Cont
 	return nil
 }
 
+func AuthenticateToOktetoCluster(ctx context.Context, c *ContextCommand, ctxOptions *ContextOptions) error {
```

Comment:
I wouldn't add this function, I think that calling a component of the context command and checking the error could be directly on the other function